<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>661</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Sentence : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_1">
        <property name="toolTip">
         <string>Введите выражение</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="submitButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Внести данные в таблицу</string>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Sample :    </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="toolTip">
         <string>Введите пример использования выражения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="refreshButton">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Выделите строку для удаления</string>
        </property>
        <property name="text">
         <string> Delete </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QToolButton" name="firstpageButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Первая страница</string>
        </property>
        <property name="text">
         <string>First Page</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image/fwd_left_arrow.png</normaloff>image/fwd_left_arrow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="popupMode">
         <enum>QToolButton::DelayedPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="prevpageButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Предыдущая страница</string>
        </property>
        <property name="text">
         <string>Prev Page</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image/left_arrow.png</normaloff>image/left_arrow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Выберите страницу</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="directpageButton">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Указанная страница</string>
        </property>
        <property name="text">
         <string>Page</string>
        </property>
        <property name="icon">
         <iconset theme="go-jump">
          <normaloff>../../Users/38050/.designer/backup</normaloff>../../Users/38050/.designer/backup</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="nextpageButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Следующая страница&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Next Page</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image/right_arrow.png</normaloff>image/right_arrow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="lastpageButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="toolTip">
         <string>Последняя страница</string>
        </property>
        <property name="text">
         <string>Last Page</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>image/fwd_right_arrow.png</normaloff>image/fwd_right_arrow.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextUnderIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_File">
   <property name="text">
    <string>New File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
